foundation/foundation-configuration-store/src/test/java/util/TestPropertyDao.java

```diff
@@ -0,0 +1,47 @@
+package util;
```

Comment:
A: We should go ahead an put this and the Init Test Helper under `com.boozallen.aissemble.util`.  Not a huge deal since it's just test code that we aren't shipping, but just a bit cleaner organization.